controls.py

```diff
@@ -144,7 +135,7 @@ def Sentence(*elems):
 
     return dbc.Form(groups)
 
-def Block(title, *body, width="100%"): #width of the block in its container
+def Block(title, *body, ison=True, width="100%"): #width of the block in its container
```

Comment:
What does `ison` mean here, and what is the use case of hiding the card in this way vs changing the style?